juriscraper/pacer/email.py

```diff
@@ -534,116 +534,55 @@ def _get_docket_entries(
         return []
 
     def _parse_bankruptcy_short_description(self, subject: str) -> str:
-        """Parse the short description of a bankruptcy case from the email
-        subject. Subjects for bankruptcy varies a lot from court to court.
-        This function supports parsing the short description for courts with
-        known examples.
+        """Parse the short description of a bankruptcy case from the email subject
+
+        The subject contains: the docket number, the case name, the short description
+        and special characters. The presence and order of these elements varies from
+        court to court. The short description that we parse out should match the
+        "Description" on the "History/Document" report on PACER
+
+        A special case are multi-docket NEFs, of which we have only seen
+        2-docket NEFs. These are labelled as "_multi_" on the example files, and we
+        have only seen "Adversary Cases", so far. The subject will use one
+        of the case names and one of the docket numbers, and then follow
+        the general rules
 
         :param subject: The email subject string.
         :return: The parsed short description.
         """
-
-        # See paeb_3.txt for a test of multi docket NEF
-        # So far, we have only seen 2-docket NEF
-        is_multidocket = len(self.docket_numbers) == 2
-        if len(self.docket_numbers) > 1 and self.court_id != "njb":
-            logger.error(
-                "Not parsing description for Bankruptcy Multi Docket NEF for court '%s'",
-                self.court_id,
-                extra={
-                    "fingerprint": [
-                        f"{self.court_id}-not-parsing-multi-docket-short-description"
-                    ]
-                },
-            )
-            return ""
+        # Some courts have subjects like
+        # `Multiple Cases "{docket} {case name} Close Adversary Case"`
+        if "close adversary case" in subject.lower():
```

Comment:
I agree with this assumption. I looked for short descriptions that contain `Close Adversary Case`, and it seems safe to return just `Close Adversary Case` in these cases. However, I think we should also consider its quoted version, `"Close Adversary Case"`, as seen in `txnb_multi_1.json`. The short description should simply be `Close adversary case`, as I confirmed in its docket history report.
![Screenshot 2024-12-23 at 6 51 17 p m](https://github.com/user-attachments/assets/7ed298c5-0ffc-488f-ab18-af42a11f8521)